### PR TITLE
🔀🌐 some first pl_PL translations

### PR DESCRIPTION
**Pull request by esem22**

### CODE REVIEW

**Comment by Piet2001** [comment]
Please change merge path to dev branch.

File: src/i18n/pl_PL.ts [hunk 1]
```diff
@@ -0,0 +1,976 @@
...
+        'ship',
+        'ship',
+        'fire',
+        'stethoscope',
+        'shield-alt',
+        'clinic-medical',
+        'plane',
+        'shield-alt',
+    ],
+    pois: [
```
Comment (JRH-1997) [inline src/i18n/pl_PL.ts:914]: Please change it to:
````
        'Park',
        'Jezioro',
        'Szpital',
        'Las',
        'Przystanek autobusowy',
        'Przystanek tramwajowy',
        'Dworzec kolejowy (ruch podmiejski)',
        'Dworzec kolejowy (ruch podmiejski i dalekobieżny)',
        'Stacja towarowa',
        'Supermarket (mały)',
        'Supermarket (duży)',
        'Stacja paliw',
        'Szkoła',
        'Muzeum',
        'Centrum handlowe',
        'Warsztat samochodowy',
        'Zjazd z autostrady',
        'Jarmark Bożonarodzeniowy',
        'Dyskoteka',
        'Stadion',
        'Gospodarstwo rolne',
        'Biurowiec',
        'Basen',
        'Przejazd kolejowy',
        'Teatr',
        'Teren wystawowy',
        'Rzeka',
        'Mały port lotniczy (pas startowy)',
        'Duży port lotniczy (pas startowy)',
        'Terminal portu lotniczego',
        'Bank',
        'Magazyn',
        'Most',
        'Bar szybkiej obsługi',
        'Cargo-port',
        'Centrum segregacji odpadów',
        'Wieżowiec',
        'Dok dla wycieczkowców',
        'Marina',
        'Przejazd kolejowo-drogowy',
        'Tunel',
        'Chłodnia składowa',
        'Elektrownia',
        'Fabryka',
        'Składowisko złomu',
        'Stacja metra',
        'Mały zbiornik na substancje chemiczne',
        'Duży zbiornik na substancje chemiczne',
        'Hotel',
        'Bar',
        'Składowisko odpadów',
        'Parking',
        'Silos',
```

File: src/i18n/pl_PL.ts [hunk 2]
```diff
@@ -0,0 +1,976 @@
...
+    },
+    globalSettings: {
+        name: 'Ustawienia główne',
+        labelInMenu: {
+            title: 'Etykieta zamiast ikony w menu',
+            description:
+                'Wyświetla prostą etykietę na pasku nawigacji zamiast logo LSSM',
+        },
+    },
+    vehicles: {
```
Comment (Piet2001) [inline src/i18n/pl_PL.ts:93]: please check this on the vehicle ID's and names.
https://github.com/LSS-Manager/lss-manager-v3/blob/6d3cae5fa634c1251470c3f23d0f3430fbc1f40c/lss-manager-v3/helperfunctions.js#L246

**Review by JRH-1997 — COMMENTED** [review]
Please change the POI and change branch to Dev. 
A lot of stuff will not work when te ID's are not correct. But you can change that later when you want.

File: src/i18n/pl_PL.ts (hunk 2 above)
Comment (Piet2001) [reply src/i18n/pl_PL.ts:93]: Also for the buildings

**Review by Piet2001 — CHANGES_REQUESTED** [review]

File: src/i18n/pl_PL.ts (hunk 2 above)
Comment (esem22) [reply src/i18n/pl_PL.ts:93]: ok

File: src/i18n/pl_PL.ts [hunk 3]
```diff
@@ -0,0 +1,976 @@
...
+        'ship',
+        'ship',
+        'fire',
+        'stethoscope',
+        'shield-alt',
+        'clinic-medical',
+        'plane',
+        'shield-alt',
+    ],
+ pois: [
```
Comment (JRH-1997) [inline src/i18n/pl_PL.ts:914]: @esem22 Why you changed it back to English?

Comment (JRH-1997) [reply src/i18n/pl_PL.ts:914]: @esem22 Please look at the [comment](https://github.com/LSS-Manager/LSSM-V.4/pull/293#discussion_r529435738) I placed before. That's the whole list:

File: src/i18n/pl_PL.ts (hunk 2 above)
Comment (JRH-1997) [reply src/i18n/pl_PL.ts:93]: The vehicle list is not correct yet. Please be sure you use the correct order, ID's and names and other needed information is correct. Vehicles with schooling needs `schooling` thats the same as what's shown ingame when a schooling is started. shownSchooling needs to be the same as the schooling in your personal list at the building.

File: src/i18n/pl_PL.ts [hunk 4]
```diff
@@ -0,0 +1,1047 @@
...
+        },
+       2: {
+            caption: 'SD',
+            color: '#990000',
+            coins: 30,
+            credits: 10_000,
+            minPersonnel: 1,
+            maxPersonnel: 1,
+            wtank: 0,
+            special: 'Wymagane po zbudowaniu 3 remiz,
```
Comment (jxn-30) [inline src/i18n/pl_PL.ts:120]: please add a `'` before the `,` in this line 😉

Comment (jxn-30) [reply src/i18n/pl_PL.ts:120]: btw same thing at the other `special` lines (130, 140, 159, 171, 211, 221, 241, 253, 265288, 320, 330, 340, 370, 382, 392)

Comment (esem22) [reply src/i18n/pl_PL.ts]: I have a question: where can I find the building ID (even from version 3)?

Comment (JRH-1997) [reply src/i18n/pl_PL.ts]: afbeelding
 The only buildings I don't know are the Alliance buildings.

Comment (jxn-30) [reply src/i18n/pl_PL.ts]: 0: "Jednostka Ratowniczo-Gaśnicza"
1: "Szkoła Pożarnicza"
2: "Stacja Pogotowia Ratunkowego"
3: "Akademia Służb Ratownictwa Medycznego"
4: "Szpital"
5: "Stacja helikopterów"
6: "Komenda Policji"
7: "Centrum Powiadamiania Ratunkowego"
8: "Akademia Policyjna"
13: "Stacja Helikopterów Policji"
14: "Miejsce koncentracji Sił i Środków"
15: "Stacja WOPR"
18: "Remiza"
19: "Posterunek Policji"
20: "Podstacja Pogotowia Ratunkowego"

Comment (esem22) [reply src/i18n/pl_PL.ts]: ok, thx

File: src/i18n/pl_PL.ts [hunk 5]
```diff
@@ -0,0 +1,1050 @@
...
+            levelcost: [
+                '1. 10.000',
+                '2. 50.000',
+                '3.-5. 100.000',
+                'Ulepszenie do JRG wynosi różnicę w cenie budynków',
+            ],
+            maxBuildings: '4.000 włącznie z JRG',
+            maxLevel: 5,
+            special:
+                ''Przy posiadaniu powyżej 24 budynków straży pożarnej koszt budowy wzrasta według wzoru:<code>100.000+200.000*LOG<sub>2</sub>(Liczba istniejących straży pożarnej â’ 22)</code>, max 1 milion kredytów. Cena za Monety pozostaje bez zmian!',
```
Comment (Piet2001) [inline src/i18n/pl_PL.ts:739]: Only one ``'`` at the beginning

**Review by Piet2001 — CHANGES_REQUESTED** [review]

File: src/i18n/pl_PL.ts [hunk 6]
```diff
@@ -0,0 +1,1050 @@
...
+            },
+            {
+                caption: 'Szkolenie SP LSP',
+                duration: '3 Dni',
+            },
+            {
+                caption: 'Szybkie ratownictwo wodne',
+                duration: '4 Dni',
+            },
+                caption: 'Kurs Nurka MSWiA',
```
Comment (Piet2001) [inline src/i18n/pl_PL.ts:900]: You're missing the opening ``{``

**Review by Piet2001 — CHANGES_REQUESTED** [review]

File: src/i18n/pl_PL.ts [hunk 7]
```diff
@@ -0,0 +1,1050 @@
...
+                duration: '3 Dni',
+            },
+
+        ],
+        'WOPR': [
+            {
+                caption: 'Szybkie ratownictwo wodne',
+                duration: '4 Dni',
+            },
+                caption: 'Kurs Nurka MSWiA',
```
Comment (Piet2001) [inline src/i18n/pl_PL.ts:940]: here also missing a ``{``

File: src/i18n/pl_PL.ts [hunk 8]
```diff
@@ -0,0 +1,1050 @@
...
+       7: {
+            caption: 'Samochód ratownictwa chemicznego',
+            color: '#990000',
+            coins: 25,
+            credits: 19_200,
+            minPersonnel: 1,
+            maxPersonnel: 1,
+            wtank: 0,
+            schooling: 'Szkoła Pożarnicza',
+            shownSchooling: 'Wymaga specjalnej edukacji (Ratownictwo chemiczne)',
```
Comment (JRH-1997) [inline src/i18n/pl_PL.ts:170]: This need to be the same as in the personal list. 
In the Dutch version is it in this example `Duiker/Duikploegleider`
afbeelding

File: src/i18n/pl_PL.ts [hunk 9]
```diff
@@ -0,0 +1,1050 @@
...
+        'WOPR': {
+            buildings: [3, 15],
+            color: '#F9D74A',
+        },
+        'Inne': {
+            buildings: [7, 4, 14],
+            color: '#02a18c',
+        },
+    },
+    vehicleCategories: {
```
Comment (JRH-1997) [inline src/i18n/pl_PL.ts:818]: Please translate the category names and vehicle groups. Did you already checked these vehicle ID's?

File: src/i18n/pl_PL.ts [hunk 10]
```diff
@@ -0,0 +1,1050 @@
...
+        'Mały zbiornik na substancje chemiczne',
+        'Duży zbiornik na substancje chemiczne',
+        'Hotel',
+        'Bar',
+        'Składowisko odpadów',
+        'Parking',
+        'Silos',
+    ],
+    only_alliance_missions: [57, 74, 89],
+    transfer_missions: [77],
```
Comment (JRH-1997) [inline src/i18n/pl_PL.ts:1051]: This one is not existing in your country. Could be an empty array.

File: src/i18n/pl_PL.ts [hunk 11]
```diff
@@ -0,0 +1,1050 @@
...
+        'Stacja metra',
+        'Mały zbiornik na substancje chemiczne',
+        'Duży zbiornik na substancje chemiczne',
+        'Hotel',
+        'Bar',
+        'Składowisko odpadów',
+        'Parking',
+        'Silos',
+    ],
+    only_alliance_missions: [57, 74, 89],
```
Comment (JRH-1997) [inline src/i18n/pl_PL.ts:1050]: This should be 57, 74

File: src/i18n/pl_PL.ts [hunk 12]
```diff
@@ -0,0 +1,1050 @@
...
+            {
+                caption: 'Szybkie ratownictwo wodne',
+                duration: '4 Dni',
+            },
+                caption: 'Kurs Nurka MSWiA',
+                duration: '5 Dni',
+            },
+        ],
+    },
+    amount: 'Quantity',
```
Comment (JRH-1997) [inline src/i18n/pl_PL.ts:947]: Please translate this and the other translations here

File: src/i18n/pl_PL.ts [hunk 13]
```diff
@@ -0,0 +1,1050 @@
...
+        7: {
+            caption: 'Centrum Powiadamiania Ratunkowego',
+            color: '#24c3ae',
+            coins: 0,
+            credits: 0,
+            extensions: [],
+            levelcost: [],
+            maxBuildings: 'Maksymalnie 1 na 25 pozostałych zbudowanych budynków',
+            maxLevel: 0,
+            special: 'The control center is the administrative center.',
```
Comment (JRH-1997) [inline src/i18n/pl_PL.ts:621]: Please translate

File: src/i18n/pl_PL.ts [hunk 14]
```diff
@@ -0,0 +1,1050 @@
...
+                '3.-5. 100.000',
+            ],
+            maxBuildings: 'Bez limitu',
+            maxLevel: 5,
+            special: 'Żeby stacja poprawnie funkcjonowała potrzebujemy do każdej łodzi pojazdu które posłuży jako laweta do holowania łodzi.',
+            startPersonnel: 10,
+            startVehicles: ['S.WOPR'],
+        },
+
+        16: {
```
Comment (JRH-1997) [inline src/i18n/pl_PL.ts:698]: Please translate this one, also extensions.

File: src/i18n/pl_PL.ts [hunk 15]
```diff
@@ -0,0 +1,1050 @@
...
+                'Cena przebudowy na Stację Pogotowania Ratunkowego wynosi różnicę w cenach budynków ',
+            ],
+            maxBuildings: 'Bez limitu',
+            maxLevel: 5,
+            special: '',
+            startPersonnel: 3,
+            startVehicles: ['Ambulans P'],
+        },
+    },
+    buildingCategories: {
```
Comment (JRH-1997) [inline src/i18n/pl_PL.ts:796]: Could you please choose the correct building ID's in the category's

File: src/i18n/pl_PL.ts [hunk 16]
```diff
@@ -0,0 +1,1050 @@
...
+            startVehicles: [],
+        },
+        13: {
+            caption: 'Stacja Helikopterów Policji',
+            color: '#148423',
+            coins: 50,
+            credits: 1_000_000,
+            extensions: [],
+            levelcost: ['1. 1.000.000 Kredytów / 50 Monet'],
+            maxBuildings: 'see specials',
```
Comment (JRH-1997) [inline src/i18n/pl_PL.ts:655]: Please translate

File: src/i18n/pl_PL.ts [hunk 17]
```diff
@@ -0,0 +1,1050 @@
...
+            startVehicles: [],
+        },
+        5: {
+            caption: 'Stacja helikopterów',
+            color: '#FFFFCC',
+            coins: 50,
+            credits: 1_000_000,
+            extensions: [],
+            levelcost: [],
+            maxBuildings: 'see specials',
```
Comment (JRH-1997) [inline src/i18n/pl_PL.ts:573]: Please translate

File: src/modules/missionHelper/i18n/pl_PL.json [hunk 18]
```diff
@@ -0,0 +1,75 @@
...
+		"rescue_helicopter_stations": "Rescue Helicopter Count",
+		"rescue_stations": "Ambulance Station | Ambulance Stations",
+		"title": "Wymagania misji | Wymagania misji"
+	},
+	"prisoners": {
+		"max_possible_prisoners": "Maximum Prisoners",
+		"title": "Prisoner | {n} Prisoner | {n} Prisoner"
+	},
+	"vehicles": {
+		"captions": {
```
Comment (JRH-1997) [inline src/modules/missionHelper/i18n/pl_PL.json:36]: I don't know if you want to translate this file directly or later?
You need to translate these captions here: 
afbeelding

File: src/modules/missionHelper/i18n/pl_PL.json [hunk 19]
```diff
@@ -0,0 +1,75 @@
...
+		"police_helicopter_stations": "Police Helicopter Station | Police Helicopter Stations",
+		"police_stations": "Police Station | Police Stations",
+		"rescue_helicopter_stations": "Rescue Helicopter Count",
+		"rescue_stations": "Ambulance Station | Ambulance Stations",
+		"title": "Wymagania misji | Wymagania misji"
+	},
+	"prisoners": {
+		"max_possible_prisoners": "Maximum Prisoners",
+		"title": "Prisoner | {n} Prisoner | {n} Prisoner"
+	},
```
Comment (JRH-1997) [inline src/modules/missionHelper/i18n/pl_PL.json:34]: Please add this here and translate: 

> 
	"requirements": {
		"water_needed": "Wasserbedarf (L)"
	},

**Review by JRH-1997 — CHANGES_REQUESTED** [review]

**Comment by jxn-30** [comment]
@esem22 @Suchty112 I would like to merge here soon. Could you guys get that ready please? :)

**Review by Suchty112 — APPROVED** [review]

**Comment by Suchty112** [comment]
@jxn-30  can be merged. even if not everything is translated.

**Pull request merged**